server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt

```diff
@@ -12,6 +12,13 @@ enum class IMUType(val id: UInt) {
 	BMI160(8u),
 	ICM20948(9u),
 	ICM42688(10u),
+	BMI270(11u),
+	LSM6DS3TRC(12u),
+	LSM6DSV(13u),
+	LSM6DSO(14u),
+	LSM6DSR(15u),
+	MPU6050_SF(16u),
```

Comment:
```suggestion
```

Reply:
I was going to ask about this:

https://github.com/SlimeVR/slimevr-node/blob/b1fd86198406f9265191528940bf008ac9b6d68b/packages/firmware-protocol/src/constants.ts#L41

I will PR the firmware repository if MPU6050_SF needs to be added.

Reply:
yes it needs to

Reply:
Im against it! It does not make sense to mix IMU with Fusion code!

Reply:
see also my comment on:
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/322#discussion_r1544627331

Reply:
Updated PR. Removed MPU6050_SF.

Also, made PR for SlimeVR-Node: https://github.com/SlimeVR/slimevr-node/pull/41